data/Language/Flow.yaml

```diff
@@ -0,0 +1,20 @@
+identifier: Flow
+full_name: Flow
+wikidata: Q59413589
+# versions: v0.1.0 - v0.87.0
```

Comment:
what is this?

Reply:
There were more than 87 releases. And it'd be a bad idea to list them all.
So just add a comment, if that's ever needed.

Reply:
Should I remove that?